diff -urN tmp/modules/image_io/pdp_qt.c pdp-0.12.4/modules/image_io/pdp_qt.c
--- tmp/modules/image_io/pdp_qt.c	2003-06-01 11:46:37.000000000 +0000
+++ pdp-0.12.4/modules/image_io/pdp_qt.c	2006-10-13 14:19:54.000000000 +0000
@@ -19,8 +19,8 @@
  */
 
 
-#include <quicktime/lqt.h>
-#include <quicktime/colormodels.h>
+#include <lqt/lqt.h>
+#include <lqt/colormodels.h>
 
 #include "pdp.h"
 #include "pdp_llconv.h"
