Author: Debian
Description: treat kFreeBSD the same as linux
Index: pdp/configure
===================================================================
--- pdp.orig/configure	2013-11-02 05:19:59.000000000 +0100
+++ pdp/configure	2013-11-02 05:19:59.000000000 +0100
@@ -3494,7 +3494,7 @@
 
 
 ARCH=`uname -s`
-if test $ARCH == Linux;
+if test $ARCH == "Linux" -o $ARCH == "GNU/kFreeBSD";
 then
 	PDP_LIBRARY_NAME=pdp.pd_linux
 
@@ -3540,7 +3540,7 @@
   cat >>confdefs.h <<_ACEOF
 #define HAVE_LIBV4L1_VIDEODEV_H 1
 _ACEOF
- have_v4l="yes"
+ have_libv4l="yes"
 fi
 
 done
@@ -3552,7 +3552,7 @@
   cat >>confdefs.h <<_ACEOF
 #define HAVE_LINUX_VIDEODEV_H 1
 _ACEOF
- have_libv4l="yes"
+ have_v4l="yes"
 fi
 
 done
Index: pdp/configure.ac
===================================================================
--- pdp.orig/configure.ac	2013-11-02 05:19:59.000000000 +0100
+++ pdp/configure.ac	2013-11-02 05:19:59.000000000 +0100
@@ -137,7 +137,7 @@
 AC_CHECK_LIB(m,sin)
 
 ARCH=`uname -s`
-if test $ARCH == Linux;
+if test $ARCH == "Linux" -o $ARCH == "GNU/kFreeBSD";
 then
 	PDP_LIBRARY_NAME=pdp.pd_linux
 
Index: pdp/system/zl/v4l.c
===================================================================
--- pdp.orig/system/zl/v4l.c	2013-11-02 05:19:59.000000000 +0100
+++ pdp/system/zl/v4l.c	2013-11-02 05:19:59.000000000 +0100
@@ -43,7 +43,9 @@
 #include <sys/ioctl.h>
 #include <sys/types.h>
 #include <sys/time.h>
+#ifdef __linux__
 #include <linux/types.h>
+#endif
 #include <stdbool.h>
 
 #include "v4l.h"
Index: pdp/opengl/Makefile.config
===================================================================
--- pdp.orig/opengl/Makefile.config	2013-11-02 05:17:57.000000000 +0100
+++ pdp/opengl/Makefile.config	2013-11-02 05:22:49.000000000 +0100
@@ -15,6 +15,14 @@
 # CPPFLAGS = -I$(PD_DIR) -I$(PDP_DIR) -I$(PDP_OGL_DIR) -I/usr/X11R6/include
 
 UNAME := $(shell uname -s)
+ifeq ($(UNAME),GNU)
+  ## GNU/Hurd behaves the same as linux
+  UNAME=Linux
+endif
+ifeq ($(UNAME),GNU/kFreeBSD)
+  ## GNU/kFreeBSD behaves the same as linux
+  UNAME=Linux
+endif
 ifeq ($(UNAME),Linux)
   TARGET=linux
   LDFLAGS = -lGL -lglut
