Author: Debian
Description: treat kFreeBSD the same as linux
--- pdp.orig/configure
+++ pdp/configure
@@ -3494,7 +3494,7 @@
 
 
 ARCH=`uname -s`
-if test $ARCH == Linux;
+if test $ARCH == "Linux" -o $ARCH == "GNU/kFreeBSD";
 then
 	PDP_LIBRARY_NAME=pdp.pd_linux
 
--- pdp.orig/configure.ac
+++ pdp/configure.ac
@@ -137,7 +137,7 @@
 AC_CHECK_LIB(m,sin)
 
 ARCH=`uname -s`
-if test $ARCH == Linux;
+if test $ARCH == "Linux" -o $ARCH == "GNU/kFreeBSD";
 then
 	PDP_LIBRARY_NAME=pdp.pd_linux
 
