Author: IOhannes m zmölnig
Descriptions: allow pdp to be compiled with hardening flags
--- pdp.orig/scaf/Makefile
+++ pdp/scaf/Makefile
@@ -26,7 +26,7 @@
 
 pdp_scaf.pd_linux: pdp_scaf_all
 	rm -f pdp_scaf.pd_linux
-	gcc -rdynamic -shared -o pdp_scaf.pd_linux pdp/*.o $(PDP_CA_LIBS)
+	gcc -rdynamic -shared -o pdp_scaf.pd_linux pdp/*.o $(PDP_CA_LIBS) $(LDFLAGS)
 
 install: all
 	install -d $(prefix)/lib/scaf
--- pdp.orig/Makefile
+++ pdp/Makefile
@@ -26,7 +26,7 @@
 
 linux: pdp_all
 	rm -f pdp.pd_linux
-	$(CC) -rdynamic -shared -o pdp.pd_linux modules/*/*.o system/pdp.o system/*/*.o puredata/*.o $(PDP_LIBS)
+	$(CC) -rdynamic -shared -o pdp.pd_linux modules/*/*.o system/pdp.o system/*/*.o puredata/*.o $(PDP_LIBS) $(LDFLAGS)
 
 linux_mmx: linux
 linux_gcc_mmx: linux
--- pdp.orig/opengl/Makefile.config
+++ pdp/opengl/Makefile.config
@@ -8,10 +8,12 @@
 
 CFLAGS = -DPD -O2 -funroll-loops -fomit-frame-pointer  -ffast-math \
     -Wall -W -Wstrict-prototypes \
-    -Wno-unused -Wno-parentheses -Wno-switch -g -I../../../../pd/src
+    -Wno-unused -Wno-parentheses -Wno-switch -g -I../../../../pd/src \
+    $(ADDITIONAL_CFLAGS)
 
 
-CPPFLAGS = -I$(PDP_DIR) -I$(PDP_OGL_DIR) -I/usr/X11R6/include -DPDP_VERSION=\"$(PDP_VERSION)\"
+CPPFLAGS = -I$(PDP_DIR) -I$(PDP_OGL_DIR) -I/usr/X11R6/include -DPDP_VERSION=\"$(PDP_VERSION)\" \
+    $(ADDITIONAL_CPPFLAGS)
 # CPPFLAGS = -I$(PD_DIR) -I$(PDP_DIR) -I$(PDP_OGL_DIR) -I/usr/X11R6/include
 
 UNAME := $(shell uname -s)
